pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1204,6 +1204,18 @@ type ObjectStoreUserSpec struct {
 	//The display name for the ceph users
 	// +optional
 	DisplayName string `json:"displayName,omitempty"`
+	//The additional capablities for the ceph user
+	// +optional
+	Caps map[string]string `json:"caps,omitempty"`
+	//The maximum bucket limit for the ceph user
+	// +optional
+	MaxBuckets int `json:"maxBuckets,omitempty"`
+	//The maximum size limit for the ceph user
+	// +optional
+	MaxSize string `json:"maxSize,omitempty"`
```

Comment:
Does this mean each user will be limited on what they can place in each bucket? Or is it a limit on the total objects across all buckets that this user has access to?

Reply:
It is total limit on the objects and size across all the buckets